include/Event.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "Packet.hpp"
```

Comment:
Where do you use  "Packet.hpp"?

Reply:
Now Event and Packet are independent classes, this include is left over from the previous Event implementation.